.circleci/config.yml

```diff
@@ -32,18 +32,40 @@ commands:
             git clone --branch ${FLUTTER_BRANCH} https://github.com/flutter/flutter.git ${FLUTTER_HOME}
             (yes || true) | flutter doctor --android-licenses && flutter doctor
 
+  install_melos:
+    steps:
+      - run:
+          name: Install and set up melos
+          command: |
+            pub global activate melos
+            melos bootstrap --ignore="*example*"
```

Comment:
Not sure if there was a reason you wanted to exclude the example apps, but we kept them in on FlutterFire as it ensures the Dart code in the examples are up to date and using the latest code from the packages. It's fairly lightweight to include these as it's only the Dart code thats part of the bootstrap.

Reply:
@Salakar Thank you so much for info! I agree that it would be a good idea to link the example apps as well. Currently I experience issues if I just run `melos bootstrap`, and it seems to be related to the `example` directory in the project root (see output below). Melos no longer produces an error if I delete or rename the `example` directory. Is this expected behavior?

```
$ melos bootstrap
   └> /Users/kyleyan/Amplify/amplify-flutter

Bootstrapping project...               SUCCESS
Linking project packages...            \Unhandled exception:
FileSystemException: Cannot open file, path = '/Users/kyleyan/Amplify/amplify-flutter/.flutter-plugins' (OS Error: No such file or directory, errno = 2)
#0      _File.open.<anonymous closure> (dart:io/file_impl.dart:356:9)
#1      _RootZone.runUnary (dart:async/zone.dart:1450:54)
#2      _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
#3      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
#4      Future._propagateToListeners (dart:async/future_impl.dart:725:32)
#5      Future._completeWithValue (dart:async/future_impl.dart:529:5)
#6      Future._asyncCompleteWithValue.<anonymous closure> (dart:async/future_impl.dart:567:7)
#7      _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
#8      _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
#9      _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
#10     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:169:5)
```

Thank you so much for your help!

Reply:
This is now fixed on the latest version (0.3.8). Turns out `flutter pub get` doesn't get flutter plugins if an example directory exists in the root, for some reason. Have pushed a fix to work around that now.

Reply:
Wow thank you so much! It is indeed working now! We are aware of the linting issues and we are working towards fixing them. Thank you!